[2024-07-08T03:03:21.560+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T03:03:21.656+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Final.create_file_task manual__2024-07-08T03:03:09.019934+00:00 [queued]>
[2024-07-08T03:03:21.708+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Final.create_file_task manual__2024-07-08T03:03:09.019934+00:00 [queued]>
[2024-07-08T03:03:21.716+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-08T03:03:21.774+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): create_file_task> on 2024-07-08 03:03:09.019934+00:00
[2024-07-08T03:03:21.796+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=67) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-08T03:03:21.802+0000] {standard_task_runner.py:63} INFO - Started process 70 to run task
[2024-07-08T03:03:21.804+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Final', 'create_file_task', 'manual__2024-07-08T03:03:09.019934+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/dag_test.py', '--cfg-path', '/tmp/tmp9re9p4lv']
[2024-07-08T03:03:21.808+0000] {standard_task_runner.py:91} INFO - Job 30: Subtask create_file_task
[2024-07-08T03:03:21.937+0000] {task_command.py:426} INFO - Running <TaskInstance: Final.create_file_task manual__2024-07-08T03:03:09.019934+00:00 [running]> on host 3b89c1ba7f0f
[2024-07-08T03:03:22.244+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ha20040204@gmail.com' AIRFLOW_CTX_DAG_OWNER='HA' AIRFLOW_CTX_DAG_ID='Final' AIRFLOW_CTX_TASK_ID='create_file_task' AIRFLOW_CTX_EXECUTION_DATE='2024-07-08T03:03:09.019934+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-08T03:03:09.019934+00:00'
[2024-07-08T03:03:22.248+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T03:03:22.251+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-08T03:03:22.259+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'touch /opt/***/share_data/database.db']
[2024-07-08T03:03:22.297+0000] {subprocess.py:86} INFO - Output:
[2024-07-08T03:03:22.315+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-08T03:03:22.316+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T03:03:22.423+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Final, task_id=create_file_task, run_id=manual__2024-07-08T03:03:09.019934+00:00, execution_date=20240708T030309, start_date=20240708T030321, end_date=20240708T030322
[2024-07-08T03:03:22.515+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T03:03:22.550+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
